common/src/main/java/jp/co/soramitsu/common/compose/component/ActionBar.kt

```diff
@@ -0,0 +1,92 @@
+package jp.co.soramitsu.common.compose.component
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import jp.co.soramitsu.common.R
+import jp.co.soramitsu.common.compose.theme.backgroundBlurColor
+import jp.co.soramitsu.common.compose.theme.white16
+import jp.co.soramitsu.common.compose.theme.FearlessTheme
+
+data class ActionItem(
+    @DrawableRes val iconId: Int,
+    val title: String,
+    val onClick: () -> Unit
```

Comment:
Don't pass onClick through the models. Pass it on to the ActionBar arguments. 
val onAction: (ActionItem) -> Unit